spec/models/chesspiece_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe ChessPiece do
+  describe "#horizontal_move?" do
+     context "given a piece at [4, 4]" do
+       let(:ChessPiece) {ChessPiece.create(position_x: 4, position_y: 4)}
```

Comment:
if you change this to `:chess_piece` it will allow you to use `chess_piece` in your `it` block like so:

```
expect(chess_piece.horizontal_move?(4, 4)).to be_falsey
```